section1/relection.md

```diff
@@ -2,18 +2,43 @@
 
 1. How did the SuperLearner Article resonate with you? What from this list do you already do? Want to start doing or do more of? Is there anything not on this list, that you would add to it?
 
+Interestingly enough I was already doing everything that was mentioned in the article.
+The only part that I thought was kind of subjective was section 6, where it's advised to take a lot of breaks and often.  My mind works a bit different.  It's hard for me to get into the flow of studying, but once I'm there I can stay focused for hours and if I stop for a break I may have a hard time getting back into that zone again.
+
+
 2. What are the data types you learned about in this section? In your own words, define each.
 
+The main focus of this section were three data types.  Strings, Numbers and Booleans.  
+Strings are a way to represent pure text without any secondary meaning (numbers can be strings when desired).  The Numbers data type is for representing digits from a mathematical standpoint.  Booleans are polarized statements that can only have the meaning of true or false.  
+
 3. How would you log the string `"Hello World!"` to the console?
 
+console.log(`"Hello World!"`);
+
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
 
+Depending on the context I would either use // for single line or /*  */ for multiline comments.
+The purpose of comments is to either provide commentary for the code being written or to make lines of code invisible to the computer.
+
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
 
+A variable is a container that can store different data types.  It's a box that can contain toys, food, paper and pens.
+
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
 
+The app that I use frequently is Google Maps.  The three variables most commonly used are probably
+var streetName, which would be a string.
+var location, which is probably a number with GPS coordinates.
+var night is probably a boolean that determine when to switch to night mode.
+
 7. In your own words, explain what concatenation is.
 
-8. Think of a site or app you use frequently. Where do you think the developers used concatention?
+Concatenation is a technique used for joining together different data types.
+
+8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
+
+When I shop on Amazon.com and want to buy several of the same item the site shows the number of items next to the item's name.
+
+9. What questions do you still have about the work we've done so far? (not having a question is not an option)
 
-9. What questions do you still have about the work we've done so far? (not having a question is not an option)
\ No newline at end of file
+When would we prefer to use Concatenation vs Interpolation?
```

Comment:
Almost never. Interpolation is much more common and usually preferable in almost every case I can think of.